ansible_collections/arista/avd/plugins/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from pathlib import Path
+
+from .plugin_utils.utils.init_logging import init_pyavd_logging
+
+PYTHON_AVD_PATH = Path(__file__).parents[4] / "python-avd"
+RUNNING_FROM_SOURCE_PATH = PYTHON_AVD_PATH / "pyavd/running_from_src.txt"
+
+init_pyavd_logging()
+
+if RUNNING_FROM_SOURCE_PATH.exists():
+    import sys
+
+    from ansible.utils.display import Display
+
+    display = Display()
+
+    display.warning(f"AVD detected it is running from source, prepending the path to the source of pyavd '{PYTHON_AVD_PATH}' to PATH to use it.")
```

Comment:
why not just use the logger here, since you just set it up?

Reply:
because this does not work despite logging being set. Need to investigate some more